### PR TITLE
fix: try002 for provider amazon

### PR DESCRIPTION
**Pull request by dondaum**
related: #38761 

fix TRY002 for provider amazon

### [amazon](https://github.com/apache/airflow/tree/main/airflow/providers/amazon)

- [x] airflow/providers/amazon/aws/operators/athena.py
- [x] airflow/providers/amazon/aws/operators/emr.py

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/providers/amazon/aws/operators/emr.py [hunk 1]
```diff
@@ -576,7 +576,7 @@ def __init__(
                 stacklevel=2,
             )
             if max_polling_attempts and max_polling_attempts != max_tries:
-                raise Exception("max_polling_attempts must be the same value as max_tries")
+                raise RuntimeError("max_polling_attempts must be the same value as max_tries")
```
Comment (o-nikolas) [inline airflow/providers/amazon/aws/operators/emr.py]: Maybe `ValueError` since we're validating the value of these arguments?

Comment (dondaum) [reply airflow/providers/amazon/aws/operators/emr.py]: Changed to `ValueError `

**Review by Taragolis — APPROVED** [review]

**Review by hussein-awala — APPROVED** [review]

**Comment by hussein-awala** [comment]
Could you merge main?

**Comment by dondaum** [comment]
> Could you merge main?

Done.

**Review by pankajkoti — APPROVED** [review]

**Pull request merged**